SimTKcommon/include/SimTKcommon/internal/Timing.h

```diff
@@ -83,8 +117,8 @@ librt realtime library (-lrt). **/
     }
 #endif
 
-#if defined(_MSC_VER) || defined(__APPLE__)
-    // On Windows and OSX, the Posix clock_gettime function is missing.
+#if defined(_MSC_VER) || SimTK_IS_APPLE_AND_MUST_DECLARE_CLOCK_GETTIME
```

Comment:
I think this is working only because `||` has lower precedence than `&&` -- seems like standing on thin ice!